impl/src/main/java/org/ehcache/internal/concurrent/ConcurrentHashMap.java

```diff
@@ -6356,7 +6358,7 @@ public final void compute() {
         return sampled;
     }
     
-    private static <T> boolean add(Set<? super T> to, Predicate<? super T> veto, T value) {
+    private static <T> boolean add(List<? super T> to, Predicate<? super T> veto, T value) {
```

Comment:
Since a list won't ever refuse to add we can probably just eliminate this method, by inlining the veto call and assuming a true return for the add.